game_eggs/doom/zandronum/egg-zandronum.json

```diff
@@ -0,0 +1,131 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2022-05-01T02:03:54+02:00",
+    "name": "Zandronum",
+    "author": "sup@libtar.de",
+    "description": "Zandronum is a multiplayer oriented port, based off Skulltag, for Doom and Doom II by id Software.\r\n\r\nThis egg uses the Freedoom WAD by default.",
+    "features": null,
+    "images": [
+        "ghcr.io\/parkervcp\/games:source"
```

Comment:
Source is desined for source games. You can use  `ghcr.io/parkervcp/yolks:debian` insteed
`